cupy/cuda/nccl.pxd

```diff
@@ -14,3 +14,18 @@ cpdef enum:
     NCCL_DOUBLE = 4
     NCCL_INT64 = 5
     NCCL_UINT64 = 6
+
+    NCCL2_INT8 = 0
```

Comment:
How about only providing new enum values?  If user uses version 1, each function convert new enum value to old enum value.